app/screens/Onboarding/Joint/Loading/LoadingScreen.data.ts

```diff
@@ -0,0 +1,15 @@
+interface ISpeechListItem {
+  title: string;
+  subtitle: string;
+}
+
+export const SPEECH_LIST: ISpeechListItem[] = [
+  {
+    title: 'screens.onboarding.joint.loading.first.title',
+    subtitle: 'screens.onboarding.joint.loading.first.subTitle',
+  },
+  {
+    title: 'screens.onboarding.joint.loading.second.title',
+    subtitle: 'screens.onboarding.joint.loading.second.subTitle',
```

Comment:
We agreed the `subTitle` -> `subtitle` change.
Please, make these changes with localisation values and don't use uppercased letters inside the localisation values